sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/MapElements.java

```diff
@@ -77,41 +73,46 @@
    */
   public static <InputT, OutputT> MapElements<InputT, OutputT> via(
```

Comment:
Same comments re: making this non-static and/or inferring the outputType from the fn.